src/iomei.cpp

```diff
@@ -7801,7 +7858,7 @@ bool MEIInput::ReadAppChildren(Object *parent, pugi::xml_node parentNode, Editor
         if (first) {
             first->SetVisibility(Visible);
         }
-        else {
+        else if (!m_deSerializing && !parent->Is(SYSTEM)) {
```

Comment:
(Minor) I would prefer the spelling `m_deserializing` as `deserializing` is a single word.

Reply:
I was hesitating. I'll change it, then.